tests/miniconda/WORKSPACE

```diff
@@ -42,6 +42,23 @@ conda_create(
     quiet = False,
 )
 
+conda_create(
+    name = "test2_env",
+    timeout = 600,
+    clean = False,
+    environment = "@//:test2_env.yml",
+    quiet = False,
+)
+
 register_toolchain(
     py3_env = "test_env",
 )
+
+# Registers a toolchain "@test2//:test2_env_toolchain".
+# The name argument is required because `register_toolchain` constructs a single external repository
+# and writes a BUILD file into it.
+register_toolchain(
+    name = "test2",
```

Comment:
This is ok but might be irritating to use. Maybe add an extra `register_toolchains` function that lets you pass in multiple envs and names?

Reply:
First thought: We can name it after the environment name, like `test2_env_toolchain`. That way only the environment name would be needed to be passed in.

Second thought: One function that registers all toolchains might be better to enforce correctness and even maybe more user-friendly.  Depends on how exactly the arguments will be passed.